apps/docs/static/example.vtt

```diff
@@ -0,0 +1,9 @@
+WEBVTT
+00:00:00.500 --> 00:00:10.000
+Hey there! 🍉 This is a sample subtitle. 👈
+00:00:10.500 --> 00:00:20.300
+This sub title is a bit longer and demonstrates the ability to render longer content onto multiple lines. It is good to have examples of this where there is a lot of content.
+00:00:21.000 --> 00:00:30.000
+We can continue to test how these subtitles might look
+00:00:30.500 --> 00:01:30.000
+This last subtitle will last until one minute and thirty seconds have passed in the video
```

Comment:
Presumably we'll need to adjust this depending on the final video file used here?

Reply:
I was thinking this could remain the same as the goal is to show both large and small subtitles. Unless you think when this comes down to an a11y review that we will want to accurately represent the video.

Reply:
Sorry, I was referring to the video length, which is 0.59s max right now but your subtitles go on until 1 minute 30s. This is a nitpick though.

Reply:
This doesn't actually affect the output which is good. Do we want to have the video actually transcribed or is this ok to keep?